src/app/containers/PageHandlers/__snapshots__/withPageWrapper.test.jsx.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`with pageWarpper should render correctly 1`] = `
```

Comment:
Hmm, this snapshot doesn't show the `<h1>Hola</h1>` anywhere 🤔

Reply:
This is an issue when using shallowRender. Will update this to use the full render